Include/genericaliasobject.h

```diff
@@ -5,6 +5,11 @@
 extern "C" {
 #endif
 
+#ifndef Py_LIMITED_API
+PyAPI_FUNC(PyObject *) _Py_apply_parameters(PyObject *, PyObject *, PyObject *, PyObject *);
```

Comment:
Sorry to trouble, but personally I didn't understand what the ``apply`` here meant. Maybe rename it to this please:
```suggestion
PyAPI_FUNC(PyObject *) _Py_subs_parameters(PyObject *, PyObject *, PyObject *, PyObject *);
```
same for the other call sites below.